code/ship/shiphit.cpp

```diff
@@ -2749,22 +2774,44 @@ static void ship_do_healing(object* ship_objp, const object* other_obj, const ve
 		return;
 	weapon_info* wip = &Weapon_info[wip_index];
 
+	float shield_health;
+	if (quadrant >= 0) {
+		shield_health = MAX(0.0f, ship_objp->shield_quadrant[quadrant] - MAX(2.0f, 0.1f * shield_get_max_quad(ship_objp)));
```

Comment:
this `0.1f *` should instead use the new `Shield_percent_skips_damage` from #6847